.github/workflows/deploy-to-eks.yaml

```diff
@@ -38,11 +46,12 @@ jobs:
           python${{ env.PYTHON_VERSION }} -m pip install tox
           sudo snap install charmcraft --classic
           # We need to install from binrary because of this https://bugs.launchpad.net/juju/+bug/2007848
```

Comment:
We should update this comment as that issue has been marked as a dup of https://bugs.launchpad.net/juju/+bug/2007575

Reply:
I believe this still applies (:

Reply:
forgot to push, updated